### ISSUE TITLE
Data analysis pipeline for soa website

### ISSUE THREAD

**Issue by theSage21** [PyJaipur/Summer-of-Algorithm#14]
The website needs a data analysis pipeline for at least the following questions. Please feel free to add your own tasks.

- [ ] data analysis on event data
    - [ ] identify participants who need help/mentorship
    - [ ] identify participants who need to be motivated
    - [ ] criteria for whom to award prizes

**Comment by arnabsinha99** [comment]
We can analyze the available slots of a mentor. In case any mentee needs help in a subject and a mentor is free at that time, the mentee's doubts can be cleared by the available mentor. 
I would like to have opinions on this.

**Comment by theSage21** [comment]
That puts the burden on the mentor to define slots no?

Perhaps we can have a generic "mentor load" which is automatically
calculated.

**Comment by theSage21** [comment]
@arnabsinha99 let's track the mentor - mentee system in #23

**Comment by arnabsinha99** [comment]
Yeah that's right. The online/offline feature would help us though. We can have a sign in feature on the website. Opinions are welcome.
